JetMETCorrections/Modules/plugins/JetCorrectorDBWriter.cc

```diff
@@ -86,9 +86,9 @@ void JetCorrectorDBWriter::beginJob() {
   if (s.isAvailable()) {
     std::cout << "Setting up payload tag " << payloadTag << std::endl;
     if (s->isNewTagRequest(payloadTag))
-      s->createNewIOV<JetCorrectorParametersCollection>(payload, s->beginOfTime(), s->endOfTime(), payloadTag);
+      s->createOneIOV<JetCorrectorParametersCollection>(payload, s->beginOfTime(), payloadTag);
     else
-      s->appendSinceTime<JetCorrectorParametersCollection>(payload, 111, payloadTag);
+      s->appendOneIOV<JetCorrectorParametersCollection>(payload, 111, payloadTag);
```

Comment:
May I ask what is the point of using a since `111` ? Is it just a placeholder instead of using `beginOfTime` or `currentTime`?

Reply:
I believe this was a placeholder for currentTime(), but after reading the slides again I realize I can work around this by replacing this section with s->writeOneIOV so I'm doing this now.